includes/blocks/safe-svg/register.php

```diff
@@ -68,10 +68,11 @@ function render_block_callback( $attributes ) {
 		'safe_svg_inline_markup',
 		sprintf(
 			'<div class="safe-svg-cover" style="text-align:%s">
-				<div class="safe-svg-inside %s" style="width: %spx; height: %spx;">%s</div>
+				<div class="safe-svg-inside %s%s" style="width: %spx; height: %spx;">%s</div>
 			</div>',
 			$attributes['alignment'] ?? 'left',
 			$class_name,
+			isset( $attributes['className'] ) ? ' ' . $attributes['className'] : '',
```

Comment:
Thanks for the PR @bmarshall511 

I noticed that none of the outputs are escaped. This line in particular can be escaped using `esc_attr()`.
I was able to inject JavaScript through the CSS classes field.

Add this string as the CSS class `"onclick="alert('hello')"` as shown in the image:

<img width="281" alt="Screenshot 2023-06-29 at 2 19 46 PM" src="https://github.com/10up/safe-svg/assets/17757960/4940c70b-3b3a-4afb-92c2-3399ea806c22">

Visit the frontend and click the SVG, observe the `alert()`.

https://github.com/10up/safe-svg/assets/17757960/bfa8b8bb-6ab9-48b8-9f4a-ce83e2174476


Let me know your thoughts on this.